### ISSUE TITLE
Having trouble replicating results

### ISSUE THREAD

**Issue by Antony-SS** [InhwanBae/ENet-SAD_Pytorch#20]
Hey, I'm using this repository for some research.  I've trained the model (the only change I made to parameters was lowering batch size to 5 for my gpu), and I'm having trouble replicating the model's results.  I am capping out at 85.8% accuracy.  Do you have a pre-trained model available?  Would really appreciate it :)
